components/MainPage/MainPageNews.vue

```diff
@@ -1,22 +1,42 @@
 <script setup lang="ts">
+import { computed } from "vue";
+
 const isMobile = useIsMobile();
+
+const { data: posts } = await useAsyncData("recent-posts", () =>
+  queryContent("/blog")
+    .where({ type: { $eq: "post" } })
```

Comment:
I forgot to update it to grab ```published: {$eq: true}```

This will still work for now but it may need to change asap for production.

Just for reference, this has to be

```
{ type: { $eq: "post" }, published: {$eq: true} }
```